tests/server/metrics-errors.js

```diff
@@ -36,18 +36,29 @@ define([
     }, dfd.reject.bind(dfd)));
   };
 
-  suite['#get /metrics-errors - returns 200 without query'] = function () {
```

Comment:
@shane-tomlinson I thought about it but it is probably not worth it. The normal DNS switch takes just a few minutes. If we miss some of the reports it's not the end of the world.